controllers/integration_test.go

```diff
@@ -522,9 +522,7 @@ var _ = Describe("Integration Test", func() {
 				By("Checking DW Directive Breakdown")
```

Comment:
Please add checking for the dbd.Status.Computes values that you expect here.

Reply:
I'll handle this in a separate PR.